docs/guides/auth.md

```diff
@@ -31,4 +31,17 @@ provider "hcp" {
   client_id     = "service-principal-key-client-id"
   client_secret = "service-principal-key-client-secret"
 }
-``` 
\ No newline at end of file
+``` 
+
+## Optional Browser Login if no client credentials configured
+
+The HCP provider supports log in via the browser if the client ID and client secret aren't configured. This feature is supported in provider version 0.45.0 and above. Enable browser login post update by first regenerating the provider in terminal with the following commands. 
+
+```
+cd terraform-provider-hcp
+make dev
```

Comment:
Question: We may need to have the user delete the existing Terraform lock file if they have it right?
```suggestion
rm .terraform.lock.hcl
make dev
```